cmd/draidcfg/.gitignore

```diff
@@ -0,0 +1 @@
+/draidcfg
```

Comment:
Looks like we can drop it because draidcfg is now absent.

Reply:
Thanks!  Will do